app/models/amr_data_feed_config.rb

```diff
@@ -2,15 +2,14 @@
 #
 # Table name: amr_data_feed_configs
 #
-#  access_type             :text             not null
```

Comment:
This was just a string - have converted to an enum now in case of future editing capabilities